src/components/Employee/Compensation/Edit.tsx

```diff
@@ -122,8 +137,8 @@ export const Edit = () => {
         label={t('amount')}
         format="currency"
         min={0}
-        errorMessage={t('validations.rate')}
-        isRequired={false}
+        errorMessage={rateErrorMessage}
+        isRequired
```

Comment:
With this enabled, can the user still submit rates being optional without the browsers default validation throwing an error here since a required field is empty?

Reply:
Yes! since we don't supply the required attribute natively. 

https://github.com/user-attachments/assets/55296dde-4c03-479c-9b53-3d7dc2eac43a